app/src/main/java/com/kickstarter/libs/utils/EventKeys.kt

```diff
@@ -0,0 +1,21 @@
+package com.kickstarter.libs.utils
+
+class EventKeys {
```

Comment:
You don't need this `EventKeys` class. This file can perfectly hold just the enum class.
The only difference that will make on `AnalyticsEvents.kt` is the imports instead of having 
`import com.kickstarter.libs.utils.EventKeys.ContextPropertyKeyName.CONTEXT_CTA` you'll get 
`import com.kickstarter.libs.utils.ContextPropertyKeyName.CONTEXT_CTA`

Reply:
Got it! The only thing is, in the future, I planned to add additional enums to this class as there are more property keys. I could remove the `EventKeys` class and put them all under one generic enum `EventKeys`, or i can keep them separate and keep the class name. Up to you! I'm fine with either one.